docs/CHANGELOG.asciidoc

```diff
@@ -142,6 +142,10 @@ The new <<mapping-ignored-field,`_ignored`>> field allows to know which fields
 got ignored at index time because of the <<ignore-malformed,`ignore_malformed`>>
 option. ({pull}30140[#29658])
 
+Plugins::
```

Comment:
I would put it in Features rather than Plugins. I don't think the fact it is exposed via a plugin matters.